porch/pkg/registry/porch/packagerevisions_approval.go

```diff
@@ -91,3 +91,17 @@ func (s packageRevisionApprovalStrategy) ValidateUpdate(ctx context.Context, obj
 }
 
 func (s packageRevisionApprovalStrategy) Canonicalize(obj runtime.Object) {}
+
+var _ SimpleRESTCreateStrategy = packageRevisionStrategy{}
```

Comment:
Should this be `var _ SimpleRESTCreateStrategy = packageRevisionApprovalStrategy{}`?

Reply:
Good call.  Fixed.